guides/source/action_controller_overview.md

```diff
@@ -899,6 +899,72 @@ Rails collects all of the parameters sent along with the request in the `params`
 [`query_parameters`]: https://api.rubyonrails.org/classes/ActionDispatch/Request.html#method-i-query_parameters
 [`request_parameters`]: https://api.rubyonrails.org/classes/ActionDispatch/Request.html#method-i-request_parameters
 
+#### `request.variant`
+
+Controllers might need to tailor a response based on context-specific
```

Comment:
When rendering to HTML I don't think the width is actually taken into account at all, it appears to combine paragraphs into a single `<p>` no matter how long the lines are.

I'm pro-80 characterss because I tend to vertically split and my editor ends up ~90 characters wide so editing guides is worse at longer line lengths.